src/registry.ts

```diff
@@ -28,7 +28,11 @@ export type FilterPairs<T> = {
 };
 
 const components = new Map<string, Entity>();
-export const registry = new ecs.World();
+
+let registry = new ecs.World();
+export function getRegistry(): ecs.World {
+	return registry;
+}
```

Comment:
It should be fine to just use `export let registry`, I don't think the function here is necessary

Reply:
The problem is query.ts caches the registry import, so every call to query never used the new updated query.

Reply:
Does it? It shouldn't

Reply:
My unit tests didn't pass until I changed this, unless I am missing something.